src/fsharp/FSharp.Core/local.fs

```diff
@@ -1,8 +1,80 @@
 // Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+
+namespace Microsoft.FSharp.Core
+
+module internal DetailedExceptions =
```

Comment:
Perhaps [<AutoOpen>] to eliminate all of the opens?  It seems like these are generally applicable.